.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,60 @@
 ## PR Summary
 
-## PR Checklist
-
-- [ ] Has Pytest style unit tests
-- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
-- [ ] New features are documented, with examples if plot related
-- [ ] Documentation is sphinx and numpydoc compliant
-- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
-- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way
-
 <!--
-Thank you so much for your PR!  To help us review your contribution, please
-consider the following points:
+Please provide at least a 1-2 sentence summary of the purpose of the PR.
+-->
 
-- A development guide is available at https://matplotlib.org/devdocs/devel/index.html.
 
-- Help with git and github is available at
-  https://matplotlib.org/devel/gitwash/development_workflow.html.
+## PR Checklists
+
+<!-- Feel free to delete any checkboxes that do not apply to this PR. -->
 
-- Do not create the PR out of master, but out of a separate branch.
+New code:
 
-- The PR title should summarize the changes, for example "Raise ValueError on
-  non-numeric input to set_xlim".  Avoid non-descriptive titles such as
-  "Addresses issue #8576".
+- [ ] Has Pytest style unit tests (and `pytest lib/matplotlib/tests` passes)
+- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check)
+- [ ] Code is documented, with examples if plot related
 
-- The summary should provide at least 1-2 sentences describing the pull request
-  in detail (Why is this change required?  What problem does it solve?) and
-  link to any relevant issues.
+New documentation:
 
+- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error)
+- [ ] Documentation conforms to matplotlib style conventions. (If you have `flake8-docstrings` and `pydocstyle<4` installed, run `flake8 --docstring-convention=all` on changed files to check).
+<!--
 - If you are contributing fixes to docstrings, please pay attention to
-  http://matplotlib.org/devel/documenting_mpl.html#formatting.  In particular,
+  http://matplotlib.org/devel/documenting_mpl.html#formatting. In particular,
   note the difference between using single backquotes, double backquotes, and
   asterisks in the markup.
+-->
+
+New features:
+
+- [ ] Have an entry in `doc/users/next_whats_new/` (follow instructions in `doc/users/next_whats_new/README.rst`)
+
+API changes:
+
+- [ ] Documented in `doc/api/api_changes_[NEXT_VERSION]` if API changed in a backward-incompatible way (follow instructions in `doc/api/api_changes_[NEXT_VERSION]/README.rst`)
+
+<!--
+Meta:
+
+- PR title descriptively summarizes the changes. (For example, prefer "Raise `ValueError` on non-numeric input to `set_xlim`" to "Addresses issue #8576").
```

Comment:
Added some copyedits directly, but think since this is a bullet list now anyway:

- PR title summarizes the changes. (Example: "Raise `ValueError` on non-numeric input `set_xlim`" )
- Please don't use non-descriptive titles such as "Addresses issue #8576"
- PR has at least a 1-2 sentence descriptive summary of how the PR implements the changes.